lib/libc/atexit.c

```diff
@@ -0,0 +1,24 @@
+#define MAX_ATEXIT 128
+
+static void (*AtExit[MAX_ATEXIT])(void);
+static unsigned NumAtExit;
+
+void klee_report_error(const char *, int, const char *, const char *);
+static void RunAtExit(void) __attribute__((destructor));
+static void RunAtExit(void) {
+    unsigned i;
+
+    for (i = NumAtExit - 1; i < MAX_ATEXIT; --i) AtExit[i]();
+}
+
+int atexit(void (*fn)(void)) {
+    if (NumAtExit == MAX_ATEXIT) {
+        klee_report_error(__FILE__, __LINE__, "atexit: no room in array!",
```

Comment:
Since it is in `libc/` folder, you should not use KLEE-specific functions. Use rather `abort`.

Reply:
Will do :+1: By the way, is it somehow different than using `klee_make_symbolic` in libc/memalloc.c? That also seems to be klee-specific.

Reply:
That is probably a bug :+1: